templates/django/docker-compose.yaml

```diff
@@ -0,0 +1,57 @@
+services:
+
+  proxy:
+    build:
+      context: ./proxy
+    restart: on-failure
+    networks:
+      - backend_network
+    depends_on:
+      - web
```

Comment:
"Web" service is not defined - docker compose throws error here. Works fine when deleted.

Reply:
I'm sort of confused why not include frontend as well.

Reply:
Because I'm working in Django directory and it kind of excludes frontend templates 😅